app/policies/user_policy.rb

```
@@ -1,45 +1,79 @@
 class UserPolicy < ApplicationPolicy
-  # Inherits user/record from ApplicationPolicy
-  # user = current_user
-  # record = target user being authorized
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
```

RuboCop (1.75.5)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)